### DISCUSSION TITLE
Problem with CLI OCR

### DISCUSSION THREAD

**Discussion by JohnDoe-JohnDoe** [cyanfish/naps2#273] [Support]
Hi,

I have been using portable version 6.1.2 for a while. I created a simple "drag n drop" BAT file to perform OCR on existing PDF's.
I have now upgraded to 7.2.2 and I can't seem to get it to work. The PDF file is updated, i.e. it's time stamp changes but the OCR has not been added. The OCR function works as expected in the GUI, just not using a command line. Has something changed?
Any help would be much appreciated.

My .BAT file

@echo off
echo start
echo
echo Adds searchable text to PDF file using the current OCR settings in NAPS2
echo

:loop
if exist "%1" (
  echo %1
  "C:\NAPS2\naps2-7.2.2-win\App\naps2.console.exe" -i %1 -o %1 --enableocr --force -n 0
  shift
  goto loop
)

pause

**Comment by cyanfish** [comment]
Looks like there's a bit of a bug with `--enableocr`, try setting `--ocrlang eng` (or [whatever](https://www.loc.gov/standards/iso639-2/php/code_list.php) language).

**Reply by JohnDoe-JohnDoe** [reply]
Many thanks for your quick reply. Yes using --ocrlang eng works great.
Cheers
JD
